web-controller/src/main/java/com/oasis/webcontroller/MappingValues.java

```diff
@@ -0,0 +1,5 @@
+package com.oasis.webcontroller;
+
+public class MappingValues {
+    public static final String LOGIN_MAPPING_VALUE = "/login";
```

Comment:
rename it to API_LOGIN
use prefix "API_"

Reply:
Commit ID: 2029415